tests/models/faraday/test_faraday_model.py

```diff
@@ -0,0 +1,72 @@
+import numpy as np
+import pytest
+import torch
+
+from opensynth.models.faraday import FaradayModel, FaradayVAE
+
+
+@pytest.fixture
+def kwh_tensor() -> torch.tensor:
+    kwh = np.random.rand(100, 48)
+    return torch.from_numpy(kwh).float()
+
+
+@pytest.fixture
+def feature_dict() -> dict[str, torch.tensor]:
+    feature_list = ["feature_1", "feature_2"]
+    output_dict: dict[str, torch.tensor] = {}
+    for feature in feature_list:
+        # numpy randint high is exclusive!
+        random_tensor = np.random.randint(low=0, high=6, size=100)
+        output_dict[feature] = torch.from_numpy(random_tensor)
+    return output_dict
```

Comment:
What happens if the feature is not an`int`?

Reply:
Not support for now - end users should subclass FaradayModel and overwrite sample_gmm/ write their own tests for custom implementation if they wish to support non-integer labels.

Will leave implementing non-integer labels into OpenSynth directly if we have a need for it ourselves.